s64da_benchmark_toolkit/reporting.py

```diff
@@ -100,7 +100,10 @@ def run_report(self, reporting_queue):
             self._check_correctness()
 
         total_runtime = self.df['timestamp_stop'].max() - self.df['timestamp_start'].min()
-        self.total_runtime_seconds = total_runtime.total_seconds()
+        if type(total_runtime) == timedelta:
+            self.total_runtime_seconds = total_runtime.total_seconds()
+        else:
+            self.total_runtime_seconds = -1
```

Comment:
Rationale?

Reply:
This probably indicates the run was broken as it stems from the ctrl-c test case, so I thought -1 would indicate sth went wrong instead of just a 0

Reply:
Hm, it somehow does not make sense to me. Why would `total_runtime` not be of type timedelta? That would mean, that further up in the calculation with `min`/`max` something went wrong, for instance if one of the fields is not a timestamp.

Reply:
Yup you can see from here: https://travis-ci.org/github/swarm64/s64da-benchmark-toolkit/builds/692100365, that its a float when the run breaks from keyboard interrupt

Reply:
I checked it, in that case, `self.df` is empty, hence this error. So, probably check whether `self.df` is empty at the very beginning. And also, this test is not 100% complete then probably. It should maybe simulate that something ran already.

Reply:
Please undo this, but introduce a check for empty `self.df` as early as possible.